src/NodeParser/TypeLiteralNodeParser.ts

```diff
@@ -93,8 +93,10 @@ export class TypeLiteralNodeParser implements SubNodeParser {
         } catch {
             // When propertyName was programmatically created, it doesn't have a source file.
             // Then, getText() will throw an error. But, for programmatically created nodes,`
-            // `escapedText` is available.
-            return (propertyName as ts.Identifier).escapedText as string;
+            // `escapedText` or `text` is available.
+            // Only `text` will be available when propertyName contains strange characters and it cannot be escaped
+            // or if it is a number.
+            return ((propertyName as ts.Identifier).escapedText as string) || (propertyName as ts.StringLiteral).text;
```

Comment:
Should this use ?? Instead so that it doesn't fall through for empty strings (if that's possible)?

Reply:
good catch, hey @baloghbence0915 can you test this out?

Reply:
Good point. Weirdly both TS and JS allows empty strings as property names.
Regardless, in such a case `escapedText` won't be available at all and will fall back to `text`, altough I aggree with the general approach of trusting `escapedText` if it present so changed OR to nullish coalescing (??) operator.
Also extended the testing with this case.